articles/stream-analytics/stream-analytics-define-inputs.md

```diff
@@ -53,6 +59,7 @@ The following table explains each property in the **New input** blade in the Azu
 | **Event hub consumer group** (optional) |The consumer group to use to ingest data from the event hub. If no consumer group is specified, the Stream Analytics job uses the default consumer group. We recommend that you use a distinct consumer group for each Stream Analytics job. |
 | **Event serialization format** |The serialization format (JSON, CSV, or Avro) of the incoming data stream. |
 | **Encoding** | UTF-8 is currently the only supported encoding format. |
+| **Compression** (optional) | The compression type (None, GZip, or Deflate) of the incoming data stream. |
```

Comment:
Since this compression setting is optional, please call out what is the default value.